modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -605,9 +605,23 @@ definition {
 	@ignore = "Test Stub"
```

Comment:
@sirfabios  please, remove the ignore, thanks!

Reply:
Done!